airflow/operators/check_operator.py

```diff
@@ -20,7 +20,7 @@
 
 import warnings
 
-from airflow.operators.sql import (
+from airflow.providers.common.sql.operators.sql import (
```

Comment:
Should the deprecation warnings in this file and others also reflect the new import path to use?

Reply:
Very much so @josh-fell :). Fixed.